### DISCUSSION TITLE
No reward yet from farming...

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#1465] [Q&A]
I ve been farming since Friday and Expected time to win was between1-2 and now its 3.
Still have 0 XCH... This farming experience is terrible and its just the start... cant imagine what will happen in a couple of weeks ahead...
Whats going on? I have checked xch_target_address and it is correct.

**Comment by dorkmo** [comment]
how many tb do you have?

**Reply by GeorgePapageorgakis** [reply]
Around 13TB

**Reply by dorkmo** [reply]
seems like you should have gotten one by now. what os and system are you farming on? Are you on gui? You might look at Last Attempted Proof

**Reply by GeorgePapageorgakis** [reply]
yes gui win10. Sometimes I see 3/ or 5/ in plots passed filter. How can i see further information on these?

**Reply by GeorgePapageorgakis** [reply]
What can I do to check if there is an issue somehow? Maybe an issue with my plots? Any scan that I can perform?

**Reply by bobochi** [reply]
I ran chia plots check -n 30.  It found a few plots that were invalid that I ended up deleting.   I'm in the same boat as you and haven't gotten any XCH and just trying to figure out if I just need to be more patient or if there's a problem lurking somewhere.  I have half the TB you have and I just seeing 1 or 2 plots passing the filter.  Hope there's a solution out there...best of luck!
